src/components/Greeting.js

```diff
@@ -0,0 +1,31 @@
+import React, { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { fetchGreeting } from '../redux/greeting/greetingSlice';
+
+const Greeting = () => {
+  const dispatch = useDispatch();
+  let greeting = useSelector((state) => state.greeting);
+
+  if (greeting === null || greeting === undefined) {
+    greeting = {};
+  }
+
+  useEffect(() => {
+    dispatch(fetchGreeting());
+  }, [dispatch]);
+
+  const postGreeting = () => {
+    if (greeting.message) {
+      return <h1>{greeting.message}</h1>;
+    }
+    return <h1>Loading...</h1>;
```

Comment:
- [ ] Your backend app returns the correct json. However, your frontend app only returns 'loading' message. Kindly fix this.

Reply:
> * [ ]  Your backend app returns the correct json. However, your frontend app only returns 'loading' message. Kindly fix this.
>   ![image](https://user-images.githubusercontent.com/91486791/222817324-5ba946f3-0aea-499e-be1e-5140c9f363f8.png)

@TadesseDev. It's not only displaying loading on my end


https://user-images.githubusercontent.com/52934128/222820721-e6328233-412e-4595-a09c-7a03f742c5d1.mp4